csv/result_test.go

```diff
@@ -30,198 +30,21 @@ type TestCase struct {
 	encoderConfig csv.ResultEncoderConfig
 }
 
-var symmetricalTestCases = []TestCase{
```

Comment:
Test cases were not being normalized from this array. Moved them into the respective test functions.